docs/data/base/components/textarea-autosize/textarea-autosize.md

```diff
@@ -51,4 +51,6 @@ Use the `minRows` prop to define the minimum height of the component:
 
 Use the `maxRows` prop to define the maximum height of the component:
 
+{{"demo": "MaxHeightTextarea.js", "noHorizontalScroll": true}}
```

Comment:
I haven't really decided if `noHorizontalScroll` is obvious enough a name

Reply:
Maybe something like `textWrap` would be a better name? Since the ultimate purpose is less about disabling the scrollbar than it is about wrapping the text.